lbz/events/api.py

```diff
@@ -46,6 +46,10 @@ def set_bus_name(self, bus_name: str) -> None:
     def sent_events(self) -> List[Event]:
         return deepcopy(self._sent_events)
 
+    @property
+    def bus_name(self) -> str:
```

Comment:
I don't mind adding this property but it won't help us solve an issue of sending events to custom event bus on demand 🤷 

As I mentioned outside GitHub, I think, we should rather consider extending the `@event_emitter` decorator and at least talk about it before implementing.

Reply:
As we agreed, the issue will be handled, at least temporarily, in the outside code.